x-pack/test/functional/apps/canvas/feature_controls/canvas_security.ts

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from 'expect.js';
+import { KibanaFunctionalTestDefaultProviders } from 'x-pack/test/types/providers';
+
+// tslint:disable no-default-export
+export default function({ getPageObjects, getService }: KibanaFunctionalTestDefaultProviders) {
```

Comment:
Thanks for defining these tests here, this is great. I'll get together a PR to move our existing functional tests for discover/dashboard appropriately.